04 webpack

davelosert opened the pull request:
@goloroden : Hier der geplante der Code der nächste Folge zum Review 🙂

goloroden commented on 04-webpack/package.json:
```diff
@@ -0,0 +1,28 @@
+  "name": "04-webpack",
+  "version": "1.0.0",
+  "description": "Kleine Einführung in die Verwendung von Webpack für React-Applikation",
+  "main": "src/index.jsx",
```
Macht das `main`-Feld hier tatsächlich Sinn? Oder kann das eventuell raus?

goloroden commented on 04-webpack/src/index.html:
```diff
@@ -0,0 +1,13 @@
+<html>
+
+<head>
+  <title>Einführung in React-Components</title>
```
Der Titel ist noch falsch, das ist der aus Folge 3 😉

davelosert commented:
@goloroden : Thanks for the feedback. I fixed the issues.

And nope, no special reason - was just in the flow. As I also used lower-case letters in the `core-compoents` folder I now switched it to lower case.
